ocfweb/lab_reservations/templates/lab_reservations/index.html

```diff
@@ -0,0 +1,92 @@
+{% extends "base.html" %}
+{% load bootstrap %}
+{% load staticfiles %}
+
+{% block content %}
+    <div class="row">
+        <div class="col-md-8 ocf-content-block">
+
+            <form method="post" action="{% url 'request_reservation' %}">
+                {% csrf_token %}
+                <p>
+                    This form is <strong>only</strong> for student groups desiring to
+                    use the OCF's lab space. If you want to create an individual account
+                    please visit <a href="{% url 'register' %}">this link</a>. If you
+                    want to request an account for a student group, please visit
+                    a staff member in the OCF lab during
+                    <a href="{% url 'staff-hours' %}">staff hours</a>.
```

Comment:
still think this is bizarre, why are you providing links to a bunch of totally unrelated forms that we already link to...?

Reply:
After further discussion with Sahil, we've decided to follow this recommendation and remove the information I provided at the top of the page.